gf-webapp/src/service/RoomApi/RoomApi.ts

```diff
@@ -1,38 +1,34 @@
 import {Currency} from "src/model/Currency";
 import {Price} from "src/model/Price";
-import {Room} from "src/model/Room/Room";
 import {roomApi, roomsApi} from "src/service/ApiInitialization";
+import {Room} from "src/autogenerated/gf-server-api/models/Room";
```

Comment:
import {Room as RoomDTO} from "src/autogenerated/gf-server-api/models/Room";

Reply:
Done

Reply:
RoomNumber function will be deleted in task #7 after change Room model